js/script.js

```diff
@@ -0,0 +1,50 @@
+var url = 'https://restcountries.eu/rest/v1/name/';
+var countriesList = $('#countries');
+
+$('#search').click(searchCountries);
+
+function searchCountries() {
+ 	var countryName = $('#country-name').val();
+  if(!countryName.length) countryName = 'Poland';
+  $.ajax({
+  		url: url + countryName,
+  		method: 'GET',
+  		success: showCountriesList
+  	});
+  }
+
+var flag = "https://restcountries.eu/data/col.svg"
+
+function showCountriesList(resp) {
+  countriesList.empty();
+  resp.forEach(function(item) {
+
+    var country = $('<li>').text(item.name);
+    country.appendTo(countriesList);
+
+    var photo = $('<p></p>');
+    var code = item.alpha3Code.toLowerCase() ; //zmiana na male litery
+    var flag = 'https://restcountries.eu/data/'+ code  + '.svg'; //adres zdjecia flagi
+    photo.html('<img class="photo" src="'+ flag +'"/>');
+    photo.appendTo(countriesList);
+
+    var capital = $('<p></p>').html("<span class=\"property\">Capital: </span> " + item.capital);
```

Comment:
instead of `class=\"property\"` use javascript to add class name

Reply:
$('#countries').find('span').addClass('property');